src/main/java/seedu/taassist/logic/parser/ParserUtil.java

```diff
@@ -140,4 +141,19 @@ public static Set<ModuleClass> parseModuleClasses(Collection<String> moduleClass
         }
         return moduleClassSet;
     }
+
+    /**
+     * Parses a {@code String moduleClass} into a {@code ModuleClass}.
```

Comment:
```suggestion
     * Parses a {@code String session} into a {@code Session}.
```
and same for line 149